libid/calcfrac.cpp

```diff
@@ -1194,14 +1194,20 @@ static void perform_worklist()
             boundary_trace();
             break;
         case 'g':
-            solid_guess();
+            if (g_calc_status != calc_status_value::COMPLETED)      // horrible cludge preventing crash when
```

Comment:
Mark the comment with TODO so we can find it as a cleanup item later.

Always use `{}` to surround statements in control blocks (e.g. `if`, `while`, etc.) like this:
```
if (g_calc_status != calc_status_value::COMPLETED)
{
    solid_guess();
}
```